lib/game_looper.rb

```diff
@@ -15,25 +18,22 @@ def loop
   attr_reader :ui, :board, :players, :outcome_checker
 
   def game_is_over
-    !outcome_checker.in_progress?
+    outcome_checker.game_over?
   end
 
   def take_turns(players)
     players.each do |player|
-      display_board
+      ui.display_board(board)
       take_turn(player) unless game_is_over
     end
   end
 
   def take_turn(player)
+    self.current_player = player.marker
```

Comment:
**suggestion:** make `current_player`'s value a consistent type

looks like it's initialized with a `Player` object but gets switched to a string here in this method. the dynamic typing allows it but it might be a bit of a surprise to someone using it.

Reply:
Thanks for the suggestion - I'll standardize the type. 

Comments like this are helping me shift my mindset from making code work to writing it for an audience - thank you!